onnxruntime/core/providers/cpu/tensor/col2im.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/providers/cpu/tensor/col2im_attributes.h"
+
+#include "core/common/common.h"
```

Comment:
just include what's needed for code in this header file, the rest can be included from the .cc file as needed